proto/select.proto

```diff
@@ -91,12 +75,32 @@ message DAGRequest {
 	optional uint64 sql_mode = 9;
 
 	// It indicates whether the sql mode is strict.
-	optional bool is_strict_sql_mode = 10;
+	// Deprecated. Don't use.
+	// optional bool is_strict_sql_mode = 10;
 
 	// supply offset is not enough since we have daylight saving time present in some regions
 	optional string time_zone_name = 11 [(gogoproto.nullable) = false];
 }
 
+// Response for DAGRequest.
+message DAGResponse {
```

Comment:
irrelevant change? Why change the name from `SelectResponse` to `DAGResponse`?

Reply:
Since this is a missing rename, leaving a pair of DAGRequest - SelectResponse. I can extract another PR to do this.